std/traits.d

```diff
@@ -4129,50 +4129,10 @@ int[] abc = cast(int[]) [ EnumMembers!E ];
 template EnumMembers(E)
 if (is(E == enum))
 {
-    import std.meta : AliasSeq;
-    // Supply the specified identifier to an constant value.
-    template WithIdentifier(string ident)
-    {
-        static if (ident == "Symbolize")
-        {
-            template Symbolize(alias value)
-            {
-                enum Symbolize = value;
-            }
-        }
-        else
-        {
-            mixin("template Symbolize(alias "~ ident ~")"
-                 ~"{"
-                     ~"alias Symbolize = "~ ident ~";"
-                 ~"}");
-        }
-    }
-
-    template EnumSpecificMembers(names...)
-    {
-        static if (names.length == 1)
-        {
-            alias EnumSpecificMembers = AliasSeq!(WithIdentifier!(names[0])
-                        .Symbolize!(__traits(getMember, E, names[0])));
-        }
-        else static if (names.length > 0)
-        {
-            alias EnumSpecificMembers =
-                AliasSeq!(
-                    WithIdentifier!(names[0])
-                        .Symbolize!(__traits(getMember, E, names[0])),
-                    EnumSpecificMembers!(names[1 .. $/2]),
-                    EnumSpecificMembers!(names[$/2..$])
-                );
-        }
-        else
-        {
-            alias EnumSpecificMembers = AliasSeq!();
-        }
-    }
+    import std.meta : Map = staticMap;
```

Comment:
Why did you keep the alias `Map` instead of using the `staticMap` directly?

Reply:
I'm trying to start a trend. :)

More seriously: `std.meta` has a lot of weird naming inconsistencies—compare `staticMap`, `Filter`, and `templateOr`, for example. I think smoothing out these inconsistencies with renamed imports makes code easier to read, and would like to see the practice catch on in Phobos. With local imports, the potential for confusion is low, since one only has to scroll a few lines to see where the alias is defined.

Reply:
Ahh, good point. I guess an alias is lightweight enough to serve this purpose. ;)

thanks. Keep the awesome work.